.github/workflows/build.yml

```diff
@@ -397,11 +397,32 @@ jobs:
           name: session-build-${{ matrix.sys.os }}
           path: build/
 
+  scan:
+    name: Scan
+    needs:
+      - os_specific
+    runs-on: ubuntu-latest
+    steps:
+      - name: Download All Build Session Artifacts
+        uses: actions/download-artifact@v2
+        with:
+          path: build/
+
+      - name: Scan for CVEs
+        if: runner.os == 'Linux'
+        uses: aquasecurity/trivy-action@0.20.0
+        with:
```

Comment:
Could you try adding `list-all-pkgs`? I'd like to see a report of the files that were scanned so we can assert that it did in fact scan the things we want it to.